examples/osggeometry/osggeometry.cpp

```diff
@@ -543,9 +545,10 @@ class MyTransformCallback : public osg::NodeCallback
 
         float               _angular_velocity;
 
-};
-
+}; //class MyTransformCallback
 
+/// Create a quad that sits behind the geometry from createScene()
+/// and shows the image from the OpenGL docs that geometry duplicates.
```

Comment:
cxw42 thanks for the clarification about /// (I did not know about it). I agree about nested /* */.
I also promote the use of // vs /* */ at work.